R/util.R

```diff
@@ -11,3 +11,13 @@ scalar <- function(x) {
 package_version_string <- function(name) {
   as.character(utils::packageVersion(name))
 }
+
+
+vcapply <- function(X, FUN, ...) { # nolint
+  vapply(X, FUN, character(1), ...)
+}
```

Comment:
```suggestion
vcapply <- function(...) {
  vapply(FUN.VALUE = character(1), ...)
}
```

I've been doing these this way recently, which avoids the nolint :)